src/index.js

```diff
@@ -429,20 +438,36 @@ module.exports = async (request, response, config = {}, methods = {}) => {
 
 	let stats = null;
 
-	try {
-		stats = await handlers.stat(absolutePath);
-	} catch (err) {
-		if (err.code !== 'ENOENT') {
-			response.statusCode = 500;
-			response.end(err.message);
+	// It's extremely important that we're doing multiple stat calls. This one
+	// right here could technically be removed, but then the program
+	// would be slower. Cause for directories, we always wanna see if a related file
```

Comment:
Nit: Proper English would be nice for code comments

Reply:
Fixed!